requirements.txt

```diff
@@ -2,15 +2,15 @@ argparse==1.2.1
 Flask==0.10.1
 SQLAlchemy==0.8.3
 gunicorn==19.4.5
-boto==2.15.0
-cmd2==0.6.7
+boto==2.45.0
+cmd2==0.6.9
```

Comment:
I would strongly recommend moving to a much newer version of cmd2.  If moving to Python 3 only, I'd recommend 0.9.12 and for Python 2 and/or both Python 2 and 3 I'd recommend 0.8.9.    

But if you use `setup.py` you can set conditional dependencies differently based upon Python version.

Reply:
Yeah I'm upgrading cmd2 as part of the creation of a testing suite.

Reply:
@deargle I'm one of the maintainers of `cmd2`.  If you run into any issues when upgrading, please feel free to create a Github Issue for us and me or one of the other devs would be happy to help.  But hopefully your experience is easy ;-)